tests/unit/source_code/samples/run_notebooks_with_fstring.py

```diff
@@ -0,0 +1,19 @@
+# Databricks notebook source
+import datetime
+# Updated List of Notebooks
+path = './leaf2.py'
+notebooks_list = [
```

Comment:
Please also add some functional tests with edge cases like resolved constant not pointing to an existing notebook

Reply:
Functional tests don't currently build dependency graphs, do we want to change that ? (we already have tests that point to missing notebooks)

Reply:
> But functional tests don't currently build dependency graphs, do we want to change that ?

yep, please add that capability :) perhaps in a separate PR then